general_superstaq/superstaq_client.py

```diff
@@ -281,11 +292,17 @@ def aqt_get_configs(self) -> Dict[str, str]:
 
     def _handle_status_codes(self, response: requests.Response) -> None:
         if response.status_code == requests.codes.unauthorized:
-            raise gss.SuperstaQException(
-                '"Not authorized" returned by SuperstaQ API.  '
-                "Check to ensure you have supplied the correct API key.",
-                response.status_code,
-            )
+            if str(response.text) == (
```

Comment:
does this work?
```suggestion
            if response.text == (
```
or does it cause a mypy issue

Reply:
Cool yeah that works, thanks.